backend/src/middleware/membersOnly.js

```diff
@@ -0,0 +1,17 @@
+/* router middleware to only allow H4I members access
+ * must be called AFTER the auth middleware
+ */
+const membersOnly = (req, res, next) => {
+  if (!req._is_member) {
+    const msg = `The key '${req._key}' doesn't have the correct privileges. Please contact the admin.`
```

Comment:
this line is causing the circle ci stuff to fail, but the pre-commit hook formats it like this so if anyone knows how to temporarily turn off the hook pls lmk !!